.github/workflows/check-query-files-and-compilation.yml

```diff
@@ -47,12 +47,12 @@ jobs:
       - name: Compile parsers Unix like
         if: matrix.os != 'windows-2022'
         run: |
-          nvim --headless -c "TSInstallSync all" -c "q"
+          nvim --headless -c "TSInstallAndQuit all" -c "sleep 180" -c "q"
```

Comment:
isn't the CI going to take too long with this sleep ?

Reply:
No. Sleep is not executed fully, when quit happens earlier.

Reply:
```
❯ time nvim --headless -c "TSInstallAndQuit jsdoc comment" -c "sleep 180" -c "q"
[nvim-treesitter] [0/2] Downloading...
[nvim-treesitter] [0/2] Downloading...
[nvim-treesitter] [0/2] Creating temporary directory
[nvim-treesitter] [0/2] Extracting...
[nvim-treesitter] [0/2] Compiling...
[nvim-treesitter] [0/2] Creating temporary directory
[nvim-treesitter] [0/2] Extracting...
[nvim-treesitter] [0/2] Compiling...
[nvim-treesitter] [1/2] Treesitter parser for comment has been installednvim --headless -c "TSInstallAndQuit jsdoc comment" -c "sleep 180" -c "q"
 0,94s user 0,23s system 96% cpu 1,214 total
```